agent/config/config.go

```diff
@@ -162,6 +169,8 @@ func Default() (*Agent, error) {
 			Address: defaultNomadAddress,
 			Region:  defaultNomadRegion,
 		},
+		APMs:    []*Plugin{{Name: defaultNomadAPM, Driver: defaultNomadAPM}},
+		Targets: []*Plugin{{Name: defaultNomadTarget, Driver: defaultNomadTarget}},
```

Comment:
Should we use something more unique for `Name` to avoid accidentally overriding the default plugins?

Reply:
Also maybe reference the values in `plugins/manager/internal.go` to avoid a mismatch if they change?